src/foam/nanos/auth/User.js

```diff
@@ -646,6 +646,8 @@ foam.CLASS({
     {
       name: 'doNotify',
       javaCode: `
+        Thread.dumpStack();
+        System.out.println("doNotify method is triggered");
```

Comment:
why not write to the log?

Reply:
you are right, I should

Reply:
no. This is during replay and logging may not yet be available.

Reply:
we are not merging this, it's just so QA has a branch to build against